python/src/equistore/tensor.py

```diff
@@ -394,6 +460,146 @@ def property_names(self) -> List[str]:
         """Names of the property labels for all blocks in this tensor map"""
         return self.block(0).properties.names
 
+    def remove_gradients(self, remove: Optional[List[str]] = None) -> None:
+        """Remove some or all of the gradients.
+
+        :param remove:
+            which gradients should be excluded from the new tensor map.
+            If this is set to ``None`` (this is the default), all the gradients will
+            be removed.
+        """
+        from equistore.operations import remove_gradients
+
+        return remove_gradients(self, remove)
+
+    def dot(self, other: "TensorMap") -> "TensorMap":
+        """Computes the dot product with another :py:class:`TensorMap`.
+
+        The given :py:class:`TensorMap` must have the same ``keys``.
+
+        :py:class:`TensorBlocks` corresponding to the same key must have the same
+        ``properties``. The resulting :py:class:`TensorBlocks` of the dot product of
+        two :py:class:`TensorBlocks` has ``result_block.values = block1.values @
+        block2.values.T``
+
+        :param other:
+            :py:class:`TensorMap` to multiply
+
+        :return: dot product
+        """
```

Comment:
I don't love that we have to duplicate the documentation here

Reply:
We can do it numpy style and briefly state.

```
"""Computes the dot product with another :py:class:`TensorMap`.

Refer to :py:func:`equistore.dot` for full documentation.
"""
```